doc/cabal-project-description-file.rst

```diff
@@ -205,8 +205,10 @@ Formally, the format is described by the following BNF:
                 | "{" Glob "," ... "," Glob "}" # union (match any of these)
 
 
-Specifying Packages from Remote Version Control Locations
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. _pkg-consume-source:
+
+Source Package Dependencies
```

Comment:
The older title was more understandable to people unfamiliar with what a source package is, but probably a bit too long. Maybe "Dependencies from Remote Version Control Systems" ?

Reply:
Changed the title to "Taking a Dependency from a Source Code Repository", a bit long but not too long.